src/app/pages/elements/select/SelectDemo.ts

```diff
@@ -42,20 +42,15 @@ export class SelectDemoComponent {
     private placeholder: string = 'Select...';
     private options: Array<string> = ['Alpha', 'Bravo', 'Charlie'];
     private withNumbers: Array<any> = [
-        { label: 'One', value: 1 },
-        { label: 'Two', value: 2 },
-        { label: 'Zero', value: 0 },
-        { label: 'Four', value: 4, readOnly: true }
```

Comment:
Does readOnly still work?

Reply:
I didn't change it, so it should. I just removed it because it didn't make sense to me to show that in the demo like that. But I feel like it does now. I'll revert that.